src/PharoLauncher-CLI-Tests/PhLCliCommandTest.class.st

```diff
@@ -81,25 +81,31 @@ PhLCliCommandTest >> testExecuteOSShellCommandWithArgs [
 ]
 
 { #category : #test }
-PhLCliCommandTest >> testFilterPrintAttributesFrom [
-
+PhLCliCommandTest >> testFilterPrintAttributesFromShouldReturnCollectionWithFirstElementWhenNoBriefFlag [
```

Comment:
this test counts with briefFlag enabled. So probably rename to testFilterPrintAttributesFromShouldReturnCollectionWithFirstElementWhenBriefFlagEnabled ?

Reply:
yes